DependencyInjection/Configuration.php

```diff
@@ -81,11 +81,17 @@ public function getConfigTreeBuilder()
                                  ->end()
                             ->end()
                         ->end()
-                        
+
                     ->end()
                 ->end()
                 ->end();
-        
+
+        $rootNode->
```

Comment:
Putting in registration node, we can override the controller for each descriminator but how do the changes in PUGXMultiUserExtension.php and services.yml ?